src/flash.sol

```diff
@@ -9,37 +9,45 @@ interface VatLike {
     function suck(address,address,uint256) external;
 }
 
-contract DssFlash is ReentrancyGuard {
+contract DssFlash {
```

Comment:
ReentrancyGuard was not defined here so it was replaced by a `lock` modifier that offers the same functionality.